helpers/recipeBook-helper.js

```diff
@@ -0,0 +1,70 @@
+const knex = require('knex')
+
+const knexConfig = {
```

Comment:
So you already have a knexfile for your config object. I would have liked to have seen your database access connection done in a third file so that you're separating concerns.